src/xpi.js

```diff
@@ -85,7 +85,7 @@ export default class Xpi {
             resolve(readStream);
           });
         })
-        .catch(err => reject(err));
+        .catch((err) => reject(err));
```

Comment:
:+1: - on payments we ended up making them optional - but in usage I've often thought I'd prefer them to be consistent.